typescript/sdk/src/models/data_source.ts

```diff
@@ -0,0 +1,40 @@
+import "reflect-metadata";
+import { DataSource } from "typeorm";
+import { SnakeNamingStrategy } from "typeorm-naming-strategies";
+import { NextVersionToProcess } from "./next_version_to_process";
+import { Base } from "./base";
+
+// TODO: Do migrations stuff??, the below function takes migrations as an argument.
```

Comment:
Yeah we need migrations, but good thing it seems like TypeORM supports this and you don't need a separate library.

Reply:
Without specifying anything it seems to just apply the migrations for us so I think we might be good here.